src/Stratis.Bitcoin.Features.MemoryPool/MempoolOrphans.cs

```diff
@@ -151,6 +150,7 @@ public class OrphanTx
         /// <returns>Whether the transaction id is present.</returns>
         public async Task<bool> AlreadyHaveAsync(uint256 trxid)
         {
+            this.mempoolLogger.LogTrace("{0}:'{1}'", nameof(trxid), trxid);
```

Comment:
Please rename mempoolLogger to logger

Reply:
Also change this line to 
`this.logger.LogTrace("({0}:'{1}')", nameof(trxid), trxid);`